pkg/processor/processor.go

```diff
@@ -178,6 +173,12 @@ func (runner *ReportRunner) Run(reportFn func(report *ReportToExecute) (map[stri
 	for _, report := range runner.Reports {
 		var err error
 
+		caseNumber, err := common.GetCaseNumberByFilename(report.File.Path)
+		if err != nil || caseNumber == "" {
```

Comment:
```suggestion
		if err != nil {
```
With your change in `common.GetCaseNumberByFilename` the second part is not
possible anymore with `err == nil`. You can remove it.

Reply:
sure can do